### PR TITLE
Increase default Briefing window size and allow custom setting

### PR DESCRIPTION
**Pull request by MjnMixael**
Not only was FRED setting the briefing icon window incorrectly, it was still trying to set it to the 640 base size. This change sets it correctly to the 1024 base size by default and adds a game_settings.tbl option to make it whatever you want, which could be useful for someone building a new UI with SCPUI and using a completely custom icon widow size game-wide.

### CODE REVIEW

**Review by Goober5000 — CHANGES_REQUESTED** [review]

File: code/mod_table/mod_table.cpp [hunk 1]
```diff
@@ -1087,6 +1111,8 @@ void mod_table_reset()
 	FS2NetD_port = 0;
 	Default_multi_object_update_level = OBJ_UPDATE_HIGH;
 	Briefing_window_FOV = 0.29375f;
+	Briefing_window_width = 888;
+	Briefing_window_height = 371;
```
[the remark this answers is not in the document]
Comment (MjnMixael) [reply code/mod_table/mod_table.cpp:1115]: Those are incorrect, as stated. If you search the code for how the brief window is rendered in-game you'll find it set to int ```
Brief_grid_coords[GR_NUM_RESOLUTIONS][4] = {
	{ // GR_640
		19, 147, 555, 232
	},
	{ // GR_1024
		30, 235, 888, 371
	}
};
```

That's top-left X/Y and then width and height for each resolution. Comparing those numbers you'll see the FRED window size was set close to the 640 size but not quite. But even more so, all that math isn't necessary here. And even more so, all that math isn't necessary here. FRED sets the window clip from top left 0,0 already so it's pointless to set, for example (width + x coordinate + 1). It makes way more sense to set it directly and exactly to what is used in-game.

File: code/mod_table/mod_table.cpp [hunk 2]
```diff
@@ -794,6 +796,28 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$Briefing Window Width:")) {
```
[the remark this answers is not in the document]
Comment (MjnMixael) [reply code/mod_table/mod_table.cpp]: Fair enough. Done as a single object.

File: code/mod_table/mod_table.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (Goober5000) [reply code/mod_table/mod_table.cpp]: Ah, good catch.

File: code/mod_table/mod_table.cpp [hunk 3]
```diff
@@ -794,6 +795,21 @@ void parse_mod_table(const char *filename)
...
 		}
 
+		if (optional_string("$FRED Briefing window resolution:")) {
+			int res[2];
+			if (stuff_int_list(res, 2) == 2) {
+				Briefing_window_resolution[0] = res[0];
+				Briefing_window_resolution[1] = res[1];
+				mprintf(("Game Settings Table: Setting FRED briefing window resolution from (%ix%i) to (%ix%i)\n",
+					res[0],
+					res[1],
```
Comment (Goober5000) [inline code/mod_table/mod_table.cpp:805]: This logging statement will show two sets of the same resolution, not the old and new resolution.  You would need to swap the logging arguments and move the assignment to after the logging.

**Review by Goober5000 — CHANGES_REQUESTED** [review]

File: code/mod_table/mod_table.cpp (hunk 3 above)
Comment (MjnMixael) [reply code/mod_table/mod_table.cpp]: Not anymore

**Review by Goober5000 — APPROVED** [review]

**Pull request merged**